script2.html

```diff
@@ -77,7 +77,7 @@
       <th scope="row">7</th>
       <td>0:04</td>
       <td>0:39</td>
-      <td>With good colors websites and applications can be easier to use in more situations</td>
+      <td>With good contrast, websites and applications can be easier to use in more situations</td>
```

Comment:
@slhenry - do we really need to remove "colors" here? Selecting good colors is the way to achieve good contrast, right? I'm concerned that "contrast" is a bit jargony and vague to many viewers.